### DISCUSSION TITLE
showing date and time picker together

### DISCUSSION THREAD

**Discussion by yunguangli** [flet-dev/flet#2905] [General]
I want to show date and time picker together in a Row or Column after only one click of a button. It seems not possible to do that in the current Flet. I tried to extend Date/Time/Container class, but to no avail.

**Comment by ndonkoHenri** [comment]
You can't. It's one or the other.
Only the [`CupertinoDatePicker`](https://flet.dev/docs/controls/cupertinodatepicker) can show both.
